docs/io/ascii/index.rst

```diff
@@ -96,6 +96,12 @@ disable the fast engine::
 
    >>> data = ascii.read(lines, format='csv', fast_reader=False)
 
+.. Note::
+
+   Reading a table which contains non-ASCII (unicode) characters is not
+   supported in Python 2.  If you need this functionality then consider
+   using a newer version of Python.
```

Comment:
Maybe worth specifying "using Python version 3 or greater" since some users (particularly Python newbies) still won't know what you mean by "a newer version".

"But I have Python 2.7.11!  That's newer than Python 2!"